js/components/ManuscriptForm.js

```diff
@@ -1,43 +1,116 @@
 import React from "react";
+
+/* ManuscriptForm - The top-level component of the manuscript/letters
+ * selection form (aka the scriptchart configuration form).
+ * Changes made to any of the form options eventually make their way
+ * back to this component's state.
+ * Its state is sent back to the App as the form data, to be
+ * processed by the DashTabs and ScriptChart components when the
+ * form is submited.
+ */
+
 import ManuscriptMenu from "./ManuscriptMenu";
 import LettersLoader from "./LettersLoader";
 
-// In this component, we would actually pass a list of
-// manuscripts, and have a toggle to sort by date or name
-// The list could come from a json object with name and date
-
 class ManuscriptForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    /* The form defaults are largely set here */
+    this.state = {
+      showBinarized: true,
+      letterExamples: 3,
+      imageSize: "Small",
+      selectedShelfmarks: [],
+      manuscripts: [],
+      letters: []
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSelect = this.handleSelect.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    const target = event.target;
+    const name = target.name;
+    const value = target.type === "checkbox" ? target.checked : target.value;
+
+    this.setState({
+      [name]: value
+    });
+  }
+
+  /* This is the handler for the multi-select items (manuscript list and letter
+   * button grid). It would be nice to integrate this with handleChagne(),
```

Comment:
Small typo in "handleChange" on this line.

Reply:
Fixed.